Library/Formula/julius.rb

```diff
@@ -0,0 +1,18 @@
+class Julius < Formula
+  homepage "http://julius.sourceforge.jp"
+  url "http://sourceforge.jp/frs/redir.php?m=iij&f=%2Fjulius%2F60273%2Fjulius-4.3.1.tar.gz"
+  sha1 "88f64ae9ed00b6ab5a2d4fe07e3ced141a46c196"
+
+  depends_on "libsndfile"
+
+  def install
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--mandir=#{man}",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+end
```

Comment:
The julius application doesn't really have a easy test method, every command it takes requires at least a language model, which can be obtained from different other projects depending on your language.

--version doesn't ekist, but --help prints out both version info and a long list of switches.